app/javascript/src/NewService/components/ServiceDiscoveryForm.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import React from 'react'
+// TODO: Get authenticityToken from rails
+// Is this needed, or was just generated by formtastic ?
+const authenticityToken = ''
+
+import type {ServiceDiscoveryFormType} from 'NewService/types'
+
+const renderSelectOptions = (options: string[]) => {
+  return options.length > 0
+    ? options.map(option => <option key={option} value={option}>{option}</option>)
+    : undefined
+}
+
+const ServiceDiscoveryForm = (props: ServiceDiscoveryFormType) => {
+  const {isVisible, providerAdminServiceDiscoveryServicesPath, namespaces, services, selectedNamespace, handleChangeNamespaces} = props
+  return (<form className={`formtastic ${isVisible ? '' : 'is-hidden'}`} id="service_discovery" action={providerAdminServiceDiscoveryServicesPath} acceptCharset="UTF-8" method="post">
```

Comment:
At the moment, we don't have a generic form component for React.
But as we are close to implement PF4-React and it includes all kind of form elements, I think we should keep this form (to match the styles of other forms generated by rails), and later replace it with PF4 forms
@josemigallas @didierofrivia @thomasmaas